transforms/universal/ededup/python/Makefile

```diff
@@ -0,0 +1,59 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+
+# $(REPOROOT)/.make.versions file contains the versions
+
+TRANSFORM_RUNTIME=python
```

Comment:
this should not be necessary.  are you finding it required?

Reply:
Its really a mess at the moment. This is cut and paste from NOOP. I looked at several of them and they are slightly different